src/contexts/globalLayout/GlobalNavbar/GalleryNavbar/GalleryNavbar.tsx

```diff
@@ -80,7 +88,7 @@ export function GalleryNavbar({ queryRef, username }: Props) {
           )}
         </NavbarCenterContent>
         <NavbarRightContent>
-          <SnowToggleIcon />
+          {isLoggedInOnMobile ? null : <SnowToggleIcon />}
```

Comment:
hide if user is logged in on mobile. this is a very rare state but there are just too many icons in this scenario